docs/requirements.md

```diff
@@ -0,0 +1,49 @@
+At its heart ServiceControl and ServicePulse provide operational support for systems built on the Particular Service Platform. They do this by:
+
+- monitoring system health
+- diagnosing and treating errors
+
+In this context, system refers to NServiceBus endpoints and communication with third-party services.
+
+ServiceControl should provide the following features:
```

Comment:
@sergioc my understanding is that "should" is used to define the vision boundaries for future SC.